src/controllers/objecting.entity.name.controller.ts

```diff
@@ -77,7 +80,20 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
     } else {
       return showPageWithSessionDataIfPresent(session, res);
     }
-    return next(new Error("No Session present"));
+  }
+  return next(new Error("No Session present"));
+}
+
+const updateMongoWithChangedUserDetails = async (session: Session,
+                                                 objectionCreate: ObjectionCreate,
+                                                 next: NextFunction) => {
+  try {
+    const company: ObjectionCompanyProfile = retrieveCompanyProfileFromObjectionSession(session);
+    const objectionId: string = retrieveFromObjectionSession(session, SESSION_OBJECTION_ID);
+    const token: string = retrieveAccessTokenFromSession(session);
+    return await updateObjectionUserDetails(company.companyNumber, objectionId, token, objectionCreate);
+  } catch (e) {
+    return next(e);
```

Comment:
need a test to check the error screen is shown for line 96

Reply:
Added a test removed the return statement.